components/renku_data_services/data_connectors/doi/schema_org.py

```diff
@@ -0,0 +1,77 @@
+"""This is used by envidat and scicat to provide information about their datasets."""
+
+from dataclasses import dataclass
+from enum import StrEnum
+from pathlib import PurePosixPath
+from urllib.parse import parse_qs, urlparse
+
+from renku_data_services.data_connectors.doi.models import SchemaOrgDataset
+from renku_data_services.errors import errors
+
+
+class DatasetProvider(StrEnum):
+    """The provider for the dataset."""
+
+    envidat = "envidat"
+
+
+@dataclass
+class S3Config:
+    """Configuration for a location on S3 storage."""
+
+    rclone_config: dict[str, str]
+    bucket: str
+    prefix: str
+
+    @property
+    def path(self) -> str:
+        """Return the path including the bucket name and the prefix."""
+        # NOTE: PurePosixPath("/test") / "/subfolder" == /subfolder, so we still have to strip /
+        return (PurePosixPath("/") / PurePosixPath(self.bucket) / self.prefix.lstrip("/")).as_posix()
+
+
+def get_rclone_config(dataset: SchemaOrgDataset, provider: DatasetProvider) -> S3Config:
+    """Parse the dataset into an rclone configuration."""
+    match provider:
+        case DatasetProvider.envidat:
+            return __get_rclone_s3_config_envidat(dataset)
+        # TODO: Add scicat here
```

Comment:
is this left on purpose?

Reply:
Yes we will add another provider Scicat from PSI soon.